src/components/form.js

```diff
@@ -47,10 +47,7 @@
         const mounted = useRef(false);
 
         B.defineFunction('Submit', () => {
-          if (formRef.current)
-            formRef.current.dispatchEvent(
-              new Event('submit', { cancelable: true }),
-            );
+          if (formRef.current) formRef.current.requestSubmit();
```

Comment:
Apparently `requestSubmit` seems not to be supported in Safari, can you check? https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/requestSubmit.